src/main/java/com/project/ita5/answer/validator/AnswerValidator.java

```diff
@@ -0,0 +1,32 @@
+package com.project.ita5.answer.validator;
+
+import com.project.ita5.answer.Answer;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import java.util.Arrays;
+
+public class AnswerValidator implements ConstraintValidator<Conditional, Answer> {
+    private String message;
+    private String[] values250 = {"1", "2", "6"};
+    private String[] values450 = {"3", "4", "5"};
+    private String[] valuesYesNo = {"3", "4", "5", "6"};
```

Comment:
what do these names mean and where are these values used?

Reply:
now fixed